packages/profile-flyout/src/ProfileFlyout.js

```diff
@@ -56,23 +57,27 @@ export default class ProfileFlyout extends Component {
     } = this.props;
 
     return (
-      <Flyout
-        alterCoordinates={alterCoordinates}
-        anchorPoint={anchorPoint}
-        content={children}
-        fallbackAnchorPoints={fallbackAnchorPoints}
-        onClickOutside={onProfileClickOutside}
-        open={open}
-        panel={panel}
-      >
-        {({ handleClick }) => (
-          <ProfileButtonPresenter
-            avatarImage={avatarImage}
-            avatarName={avatarName}
-            onClick={combineEventHandlers(handleClick, onProfileImageClick)}
-          />
+      <ThemeContext.Consumer>
+        {() => (
+          <Flyout
+            alterCoordinates={alterCoordinates}
```

Comment:
I originally wasn't even going to add `ThemeContext.Consumer` as it isn't necessary for this component since all the components that make up the `profile-flyout` are already themeable, but figure I would add this in for consistency and in case there are specific parts of `profile-flyout` that will use `theme-data` later on.